### DISCUSSION TITLE
Java JSON Schema Support

### DISCUSSION THREAD

**Discussion by KylinShinlyLin** [json-schema-org/community#785] [Q&A]
![image](https://github.com/user-attachments/assets/0a0128d2-f5f8-4d20-95c2-1bcc76977a53)
Lately, while using Java to integrate with OpenAI API's latest features, I noticed there’s a special field called additionalProperties in their official JSON schema. However, I found that Java's JSON schema tools don't generate this particular field. Any tips on making Java support this format?

**Comment by gregsdennis** [comment]
Hey @KylinShinlyLin.  Thanks for the question.  I think you're going to get the best answers from the maintainers of that particular tool, which likely means going to their repo.

We can answer questions about JSON Schema in general, but less so about specific tooling (unless the maintainer hangs around here).

For information about the keyword itself, please see [our docs](https://json-schema.org/understanding-json-schema/reference/object#additionalproperties).

**Reply by KylinShinlyLin** [reply]
Got it, thanks
